src/mol-canvas3d/passes/postprocessing.ts

```diff
@@ -226,7 +226,7 @@ function getPostprocessingRenderable(ctx: WebGLContext, colorTexture: Texture, d
 
         uMaxPossibleViewZDiff: ValueCell.create(0.5),
 
-        dOcclusionEnable: ValueCell.create(false),
+        dOcclusionEnable: ValueCell.create(true),
```

Comment:
Why this change?

Reply:
That is just to bring it in line with the default params of the pass so the shader does not get re-created right after initialization. Btter, of course, would be to support passing the params but did not get to that.